contrib/runners/go_runner/go_runner.py

```diff
@@ -0,0 +1,380 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from os.path import basename
+import sys
+import json
+import uuid
+from subprocess import list2cmdline
+
+from eventlet.green import subprocess
+
+from st2common import log as logging
+from st2common.runners.base import ActionRunner
+from st2common.util.green.shell import run_command
+from st2common.constants.action import ACTION_OUTPUT_RESULT_DELIMITER
+from st2common.constants.action import LIVEACTION_STATUS_SUCCEEDED
+from st2common.constants.action import LIVEACTION_STATUS_FAILED
+from st2common.constants.action import LIVEACTION_STATUS_TIMED_OUT
+from st2common.constants.runners import GO_RUNNER_INVALID_ACTION_STATUS_EXIT_CODE
```

Comment:
There's no strict requirement that runner related constants go back inside `st2common`. It might make more sense to have a runner specific constants file to import from to keep all runner related code in one place.